pkg/yurttunnel/server/anpserver.go

```diff
@@ -146,6 +148,7 @@ func runMasterServer(handler http.Handler,
 			ReadTimeout:  constants.YurttunnelANPInterceptorReadTimeoutSec * time.Second,
 			TLSConfig:    tlsCfg,
 			TLSNextProto: make(map[string]func(*http.Server, *tls.Conn, http.Handler)),
+			ErrorLog:     log.New(NewHandshakeFilterWriter(os.Stderr), "", log.LstdFlags),
```

Comment:
Why do we need to filter out the handshake error log? 
It will make debug a bit more difficult, won't it?

Reply:
@Congrool some healthy checkers use tcp protocol to  check tunnel-server 10263 port status but do not start tls handshake, so a large amount `tls handshake error` logs are dumped in tunnel-server, this logs will confused end user to debug `tunnel-server` problems. at the same time, if tunnel-agent fails to connect `tunnel-server` for `tls handshake error`, tunnel-agent side also will get enough log message.

Reply:
OK, but I am still confused about this scenario. Is it possible to solve this problem through opening a new port for health check instead of using 10263?
By the way, in my mind, 10263 is the port to handle dnat requests from apiserver. The `tunnel-agent` uses port 10262 to connect to the `tunnel-server`. https://github.com/openyurtio/openyurt/blob/master/cmd/yurt-tunnel-server/app/options/options.go#L69